Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [workspace]
 members = ["crates/*"]
-exclude = ["crates/test", "crates/test-streams"]
+exclude = ["crates/demo", "crates/test-streams"]
```

Comment:
@ray-kast what do you think about have our examples in `examples`?

```
examples/
  simple
  streams
crates/
  core
  runtime
```

```suggestion
exclude = ["examples/*"]
```

Reply:
yeah this is ideal makes sense

Reply:
@Nagaprasadvr make this change please

Reply:
Will do